crates/solidity/testing/snapshots/cst_output/UsingDirective/destructure_empty/input.sol

```diff
@@ -0,0 +1 @@
+using {} for *;
```

Comment:
nit: suggest adding the helpful comment like the others:

```suggestion
// empty symbols list is not valid.
using {} for *;
```

Reply:
Sure